requirements.txt

```diff
@@ -5,3 +5,4 @@ six
 # wyzesense2mqtt
```

Comment:
The requirements.txt file also needs `docopt`

Reply:
Nice catch. The bridge_tool_script is basically a slightly modified sample.py from WyzeSensePy, so I hadn't done much testing there beyond making sure the fix routine for the bad MAC was working. Fixed in next commit group.